unittests/test_using_virtual_board/test_introlabs.py

```diff
@@ -55,8 +55,13 @@ def check_directory(self, path):
                 try:
                     self.check_script(script)
                 except Exception as ex:
-                    print("Error on {}".format(script))
-                    raise ex
+                    if "virtual machine" in str(ex):
```

Comment:
WTF!?

Reply:
This is a copy of:
https://github.com/SpiNNakerManchester/sPyNNaker8/blob/master/unittests/test_using_virtual_board/test_examples.py#L68-L78

I think this is needed just in case someone adds a script that reads connections, where the connections are generated on the machine, but a virtual machine is in use.  It covers a corner case; maybe the thing it was protecting has been removed?

Reply:
if that was the case, shouldnt that be a separate integration test in spynnaker8? what its doing in test_introlabs?

Reply:
This is not currently in master, so maybe it isn't needed since master works

Reply:
id agree

Reply:
This has been removed; instead the synapses are now generated on the machine for definite if a virtual machine is in use.